drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -298,7 +284,28 @@ static int32_t optimize_regulator_voltage_scale(uint32_t sysclk_freq)
 	defined(CONFIG_CLOCK_STM32_PLL_SRC_HSI) || \
 	defined(CONFIG_CLOCK_STM32_PLL_SRC_CSI)
 
-static int32_t get_vco_output_range(uint32_t vco_input_range)
+static int get_vco_input_range(uint32_t m_div, uint32_t *range)
+{
+	uint32_t vco_freq;
+
+	vco_freq = PLLSRC_FREQ / m_div;
```

Comment:
```suggestion
	const uint32_t vco_freq = PLLSRC_FREQ / m_div;
```

Not sure about `const`, maybe it need to be `static const`. Anyway the idea behind of this change - give compiler a chance to optimize better.

Reply:
this is a stack variable, there's no need to do that